packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -44,7 +44,7 @@ const enum PlatformName {
   ios = 'iOS'
 }
 
-interface PreviewQuickPickItem extends vscode.QuickPickItem {
+export interface PreviewQuickPickItem extends vscode.QuickPickItem {
```

Comment:
Let's use PlatformName to pass to getRememberedDevice/updateRememberedDevice? `PreviewQuickPickItem` is UI specific. I guess the enum comment is making more sense now :)

Reply:
Good call.  Yes, much cleaner this way.